src/App.tsx

```diff
@@ -15,48 +16,119 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabet = 'Sort alphabetically',
+  Length = 'Sort by length',
+}
+
+type Button = {
+  name: SortType | 'Reverse',
+  color: 'is-info' | 'is-success' | 'is-warning',
+};
+
+const buttons: Button[] = [
+  {
+    name: SortType.Alphabet,
+    color: 'is-info',
+  },
+  {
+    name: SortType.Length,
+    color: 'is-success',
+  },
+  {
+    name: 'Reverse',
+    color: 'is-warning',
+  },
+];
+
+let goods = [...goodsFromServer];
+
+function operateGoods(stateSort: string | null, stateReverse: string | null) {
+  switch (stateSort) {
+    case SortType.Alphabet:
+      goods.sort((good1, good2) => good1.localeCompare(good2));
+      break;
+    case SortType.Length:
+      goods.sort((good1, good2) => good1.length - good2.length);
+      break;
+    default:
+      break;
+  }
+
+  if (stateReverse) {
+    goods = goods.reverse();
+  }
+
+  return '';
```

Comment:
```suggestion
  return goods;
```

Reply:
[comment](https://github.com/mate-academy/react_list-of-goods-hooks/pull/1139/files#r1263490958) not fixed